meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -310,6 +313,12 @@ do_install() {
     install -d ${D}${sysconfdir}/udev/mount.blacklist.d
     echo ${MENDER_ROOTFS_PART_A} > ${D}${sysconfdir}/udev/mount.blacklist.d/mender
     echo ${MENDER_ROOTFS_PART_B} >> ${D}${sysconfdir}/udev/mount.blacklist.d/mender
+
+    # install the D-Bus interface file
+    if [ -f ${B}/src/${GO_IMPORT}/Documentation/io.mender.Authentication1.xml ]; then
+        install -d ${D}${datadir}/dbus-1/interface
+        install -m 0644 ${B}/src/${GO_IMPORT}/Documentation/io.mender.Authentication1.xml ${D}${datadir}/dbus-1/interface
+    fi
```

Comment:
I think we should be future ready here and install all `Documentation/io.mender.*.xml` files. In very short time we are having here an extra file for `Update1` interface,